IFC4x3/Sections/Resource definition data schemas/Schemas/IfcRepresentationResource/Entities/IfcGeographicCRS/Documentation.md

```diff
@@ -0,0 +1 @@
+_IfcGeographicCRS_ is a coordinate reference system that is based on a ellipsoid. Geographic CRS typically requires two angles and a height to determine a unique position.
```

Comment:
A geographic coordinate reference system is a CRS that uses a three-dimensional spherical surface to determine locations on the Earth. Any location on Earth can be referenced by a point with longitude and latitude coordinates.